### DISCUSSION TITLE
Components don't show anymore

### DISCUSSION THREAD

**Discussion by BlockChainCaffe** [quasarframework/quasar#13529] [General - Components / Directives / etc]
Hi Everyone,
I'm looking for help on a very strange thing and I don't even know where to start....
Vue components stopped showing in my app, and in the code inspect, now i see all the <q-something> code.
It's like it's not compiled and is interpreted as wrong html code... or something.
Any hints?

**Comment by MilosPaunovic** [comment]
Without more info it's impossible to help. Provide a reproduction of some kind, error log, somehing...

**Reply by BlockChainCaffe** [reply]
Thanks, but that's part of the problem
I have no errors !!

But if I inspect the code I see this : 

```
<q-input id="qEmail" class="q-mb-xl" modelvalue="" modelmodifiers="[object Object]" rules="function (d) {
      return $data.email_regexp.test(d) || 'Email non conforme';
    }" outlined="" type="email" hint="Email" label="Inserisci l'email"></q-input>

```
while in the working version the q-input is translated into labels etc....

How can I search/find more info about what's going on?
TIA

**Reply by MilosPaunovic** [reply]
Step 1: Open browser console; look for errors.
Step 2: Try to add only simple input, like any from the docs page, to check if it related to the rest of your code or Quasar itself. If that works, and it's something from your code, we're gonna need reproduction for anyone to be able to help you.

**Reply by BlockChainCaffe** [reply]
I managed to solve it like this:
- checked out the project in a new folder
- checked out a previous commit
- run npm install and then nom run serve
... and it worked.
Turns out it was due to a 'npm audit' / 'npm audit --force'
Seems like there is some huge compatibility issue somewhere....

Thanks a lot
